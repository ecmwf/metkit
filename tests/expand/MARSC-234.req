retrieve,
area=Europe,class=od,date=2023-12-20,expver=1,grid=0.125/0.125,levtype=sfc,param=22.228,step=0,stream=oper,time=12:00:00,type=fc,target=target.27f72f51-1e60-437f-b1ce-e57129e2dff6,
target="reference.A0FgVE.data"