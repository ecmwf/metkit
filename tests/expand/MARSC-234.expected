RETRIEVE,
    CLASS      = OD,
    TYPE       = FC,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 228022,
    DATE       = 20231220,
    TIME       = 1200,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = reference.A0FgVE.data,
    RESOL      = AUTO,
    AREA       = 73.5/-27/33/45,
    GRID       = .125/.125 